deps.edn

```diff
@@ -34,8 +34,10 @@
 
            :sql       {:extra-paths ["src/sql" "src/sql-tests"]
                        :extra-deps  {com.h2database/h2              {:mvn/version "1.4.199" :exclusions [org.slf4j/slf4j-nop]}
+                                     seancorfield/next.jdbc         {:mvn/version "1.1.613"}
```

Comment:
This should not be here. If it is missing from rad-sql, then it should be added there.